app/assets/stylesheets/arclight/modules/sidebar.scss

```diff
@@ -1,3 +1,12 @@
 .al-in-person-repository-location {
   clear: left;
 }
+
+.al-digital-object {
+  margin-bottom: $spacer;
+}
+
+// Hiding this from display since we don't need it.
+dt.blacklight-digital_objects_ssm {
+  display: none;
```

Comment:
I'd say go with this for now and at some point soon I'll look at the page again from an accessibility perspective and adjust if necessary.